daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/ProcessorStateBases.scala

```diff
@@ -119,8 +122,7 @@ trait SetProcessorMixin {
  * which should be isolated to the alternative parser, and repParsers, i.e.,
  * places where points-of-uncertainty are handled.
  */
-abstract class ParseOrUnparseState protected (
-  protected var variableBox: VariableBox,
+abstract class ParseOrUnparseState protected (protected var variableBox: VariableBox,
   var diagnostics: List[Diagnostic],
```

Comment:
Yup. I see an eclipse setting that is forcing this. 

There are two default settings in Eclipse that do this stuff "wrong" (for now) in my view. 
One says each parameter declared in a method or class goes on a new line, even the first. 
(Presumably not if there is only one or if they all fit one line ?? Unsure)
The other says each parameter passed in a call goes on a new line, even the first.
(Presumably not if there is only one, or if they all fit one line ?? Unsure)
 
So long as they leave one-liners alone those settings are nice I think. But I wish there was a setting where I could say correct indentation only on lines I edit. 

For now I have unchecked both boxes, which should not do these unneeded whitespace tweeks for now. 




What I don't like is that I can't tell eclipse - fix things I've edited, otherwise leave them alone.

Reply:
You asked what style we "should" use. I think we should use the default standard indentation format for scala. One would hope all tools would default to this, but I'm not sure they agree on it.